### ISSUE TITLE
[example] provide example container that accepts inputs for user to test

### ISSUE THREAD

**Issue by vsoch** [expfactory/expfactory#74]
This can be an actual container, or a recipe for one. The updated "format" so far looks like:

```
exp_id  var_name    var_value         token
stroop  color       red               123
stroop  color       blue              456
stroop  words       eat,the,thing     123
stroop  words       omg,tacos         456
```

**Comment by vsoch** [comment]
should be derived from here https://github.com/expfactory-experiments/test-parse-url

**Comment by vsoch** [comment]
This will be closed with #75

**Comment by vsoch** [comment]
Documentation (overview) --> https://expfactory.github.io/expfactory/usage#participant-variables
Complete detailed container / walk through --> https://github.com/expfactory-experiments/test-parse-url/tree/master/docker
